bigbluebutton/bigbluebutton
# fix lock settings

This PR supersedes PR 2779.

To test:
1. Using API mate, join as moderator and viewer.
2. Have the viewer join audio and share webcam.
3. On moderator client, click on gear icon and choose Lock Viewers. Lock webcam, mic, chat then click apply.
4. Viewer should be muted, stopped sharing cam, chat disabled.
